docs/configuration.md

```diff
@@ -2461,9 +2461,10 @@ Apart from these, the following properties are also available, and may be useful
   <td><code>spark.task.maxFailures</code></td>
   <td>4</td>
   <td>
-    Number of failures of any particular task before giving up on the job.
+    Number of continuous failures of any particular task before giving up on the job.
     The total number of failures spread across different tasks will not cause the job
-    to fail; a particular task has to fail this number of attempts.
+    to fail; a particular task has to fail this number of attempts continuously.
+    Be aware that if one of attempts succeed, current count of task failures will be reset.
```

Comment:
nit: `Be aware that if one of attempts succeed, current count of task failures will be reset` -> `If any attempt succeeds, the failure count for the partition will be reset`

Reply:
Got that，thanks